### PR TITLE
Move Detrav recipes to coremod

### PR DESCRIPTION
**Pull request by ah-OOG-ah**
As the title describes. While it might not cause issues, this should probably wait on the [Detrav PR](https://github.com/GTNewHorizons/DetravScannerMod/pull/30).

### CODE REVIEW

File: src/main/java/com/dreammaster/main/XMod.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.dreammaster.main;
+
+import cpw.mods.fml.common.Loader;
+
+public class XMod {
+
+    public static final boolean DETRAVSCANNERMOD = Loader.isModLoaded("detravscannermod");
```
Comment (boubou19) [inline src/main/java/com/dreammaster/main/XMod.java:7]: Isn't detrav in the GT enum?

Comment (ah-OOG-ah) [reply src/main/java/com/dreammaster/main/XMod.java:7]: nope

Comment (ah-OOG-ah) [reply src/main/java/com/dreammaster/main/XMod.java:7]: I can make a PR there if you want though

Comment (ah-OOG-ah) [reply src/main/java/com/dreammaster/main/XMod.java:7]: [PR out](https://github.com/GTNewHorizons/GT5-Unofficial/pull/2663)

Comment (boubou19) [reply src/main/java/com/dreammaster/main/XMod.java:7]: merged

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**